### PR TITLE
[Convection-Diffusion] Warning of deprecated FindPointOnMesh removed

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by loumalouomega** [comment]
Anyone?, this is just a simple clean up to reduce warnings

**Comment by loumalouomega** [comment]
Ping

File: applications/convection_diffusion_application/custom_utilities/bfecc_convection.h [hunk 1]
```diff
@@ -266,58 +266,58 @@ template<std::size_t TDim> class BFECCConvection
...
-				    inode->GetSolutionStepValue(rVar,1) = inode->FastGetSolutionStepValue(rVar);
-				}
-			}
-			KRATOS_CATCH("")
+                KRATOS_TRY
+
+                ModelPart::NodesContainerType::iterator inodebegin = rModelPart.NodesBegin();
+                vector<unsigned int> node_partition;
+                #ifdef _OPENMP
+                    int number_of_threads = omp_get_max_threads();
```
Comment (RiccardoRossi) [inline applications/convection_diffusion_application/custom_utilities/bfecc_convection.h:276]: ok, however here it would have been best to use omp utils

**Review by RiccardoRossi — APPROVED** [review]

File: applications/convection_diffusion_application/custom_utilities/bfecc_convection.h (hunk 1 above)
Comment (loumalouomega) [reply applications/convection_diffusion_application/custom_utilities/bfecc_convection.h:276]: It was already there, I just changed tabulation

**Pull request merged**